### ISSUE TITLE
Allowing vec of keywords for class declaration (feature req)

### ISSUE THREAD

**Issue by code-ape** [reagent-project/reagent#245]
I've noticed that classes currently can be declared using:

1) Strings

``` clojure
[:p {:class "foo"}]
```

2) Keywords

``` clojure
[:p {:class :foo}]
```

3) A vec with a single keyword:

``` clojure
[:p {:class [:foo]}]
```

4) However a vec with multiple keywords isn't supported. Currently this:

``` clojure
[:p {:class [:foo :bar]}]
```

Results in:

``` html
<p class="foo,bar">
```

I think it would be nifty, and more composable, to have this fourth option. Thoughts? I'm happy to make the changes to the library 😃

Thanks,
codeape

**Comment by mike-thompson-day8** [comment]
This is a much request feature. Duplicate of #154.   I'd very much like to get this on the agenda for the next release. But I wouldn't want to delay 0.6.0 any further.

**Comment by olivergeorge** [comment]
We migrated from sablono to reagent and bumped into this.  With 22k loc checking all cases was tricky so we've added a spec to `tmpl/convert-props` to catch the problem cases at dev time.

**Comment by Deraen** [comment]
Duplicate of #154, merged #138 and released as 0.8.0-alpha2.
